File: leetcode-questions/csharp/src/test/csharp/com/leetcode/questions/theskylineproblem/TheSkylineProblemTest.cs
using Xunit;
using System.Collections.Generic;
using com.leetcode.questions.leetcode.editor.en.theskylineproblem;
using com.leetcode.questions.support;

namespace com.leetcode.questions.theskylineproblem;

public class TheSkylineProblemTest {

    private const string CasesFileName = "the-skyline-problem.json";

    public TheSkylineProblemTest() {

    }

    [Fact]
    public void TestCases() {
        GeneratedLeetCodeTestHarness.Run(typeof(Solution), "GetSkyline", GeneratedLeetCodeTestHarness.ReadCasesFile(CasesFileName));
    }
}

File: leetcode-questions/java/src/test/java/com/leetcode/questions/theskylineproblem/TheSkylineProblemTest.java
package com.leetcode.questions.theskylineproblem;

import com.leetcode.questions.leetcode.editor.en.TheSkylineProblem;
import com.leetcode.questions.support.GeneratedLeetCodeTestHarness;
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;
import java.nio.file.Files;
import java.nio.file.Path;

public class TheSkylineProblemTest {

    private static final Path CASES_PATH = Path.of("generated-test-cases", "the-skyline-problem.json");

    public TheSkylineProblemTest() {

    }

    @Test
    public void testCases() throws Exception {
        GeneratedLeetCodeTestHarness.run(TheSkylineProblem.Solution.class, "solve", Files.readString(CASES_PATH));
    }
}
